packages/core/src/tools/wrap-hydrate.js

```diff
@@ -4,9 +4,10 @@ const crypto = require('crypto');
 
 const { DehydrateError } = require('../errors');
 
-// Max URL length for Node is 8KB (https://stackoverflow.com/a/56954244/)
-// 8 * 1024 * (3 / 4) = 6144 max, minus some room for additional overhead
-const MAX_PAYLOAD_SIZE = 6000;
+// https://nodejs.org/docs/latest-v16.x/api/http.html#httpmaxheadersize
+// base64 encoding is approx 4/3, so our limit should be:
```

Comment:
I appreciate adding this context for where the 3/4 comes from!
```suggestion
// base64 encoding is approx 3/4, so our limit should be:
```

Reply:
I think the `4/3` is correct, if a str is 100 chars, it increases the length by 4/3 (+ 33%) = 133 chars
To find the limit of the original value
if the limit is 100, we have to take the inverse to 3/4 = 75 chars 
^ but I can make that more clear!